foa-site.php

```diff
@@ -1301,72 +1301,41 @@ function instagram_items_to_html($instagram_items){
 			$insta_array = json_decode( $insta_json[0], TRUE );
 			if ( !$insta_array )
 				throw new Exception('Instagram has returned invalid data.');
-			// old style
-			if ( isset( $insta_array['entry_data']['UserProfile'][0]['userMedia'] ) ) {
-				$images = $insta_array['entry_data']['UserProfile'][0]['userMedia'];
-				$type = 'old';
-			// new style
-			} else if ( isset( $insta_array['entry_data']['ProfilePage'][0]['user']['media']['nodes'] ) ) {
-				$images = $insta_array['entry_data']['ProfilePage'][0]['user']['media']['nodes'];
-				$type = 'new';
+			if ( isset( $insta_array['entry_data']['ProfilePage'][0]['graphql']['user']['edge_owner_to_timeline_media']['edges'] ) ) {
+				$images = $insta_array['entry_data']['ProfilePage'][0]['graphql']['user']['edge_owner_to_timeline_media']['edges'];
 			} else {
 				throw new Exception('Instagram has returned invalid data.');
 			}
 			if ( !is_array( $images ) )
 				throw new Exception('Instagram has returned invalid data.');
 			$instagram = array();
-			switch ( $type ) {
-				case 'old':
-					foreach ( $images as $image ) {
-						if ( $image['user']['username'] == $username ) {
-							$image['link']						  = preg_replace( "/^http:/i", "", $image['link'] );
-							$image['images']['thumbnail']		   = preg_replace( "/^http:/i", "", $image['images']['thumbnail'] );
-							$image['images']['standard_resolution'] = preg_replace( "/^http:/i", "", $image['images']['standard_resolution'] );
-							$image['images']['low_resolution']	  = preg_replace( "/^http:/i", "", $image['images']['low_resolution'] );
-							$instagram[] = array(
-								$image['caption']['text'],
-								'link'		  	=> $image['link'],
-								'time'		  	=> $image['created_time'],
-								'comments'	  	=> $image['comments']['count'],
-								'likes'		 	=> $image['likes']['count'],
-								'thumbnail'	 	=> $image['images']['thumbnail'],
-								'large'		 	=> $image['images']['standard_resolution'],
-								'small'		 	=> $image['images']['low_resolution'],
-								'type'		  	=> $image['type']
-							);
-						}
-					}
-				break;
-				default:
-					foreach ( $images as $image ) {
-						$image['display_src'] = preg_replace( "/^http:/i", "", $image['display_src'] );
-						if ( $image['is_video']  == true ) {
-							$type = 'video';
-						} else {
-							$type = 'image';
-						}
-						$instagram[] = array(
-							'description'           => __( $image['caption'], 'wpiw' ),
-							'link'		  	=> '//instagram.com/p/' . $image['code'],
-							'time'		  	=> $image['date'],
-							'comments'	  	=> $image['comments']['count'],
-							'likes'		 	=> $image['likes']['count'],
-							'thumbnail'	 	=> $image['display_src'],
-							'type'		  	=> $type
-						);
-					} 
-				break;
-			}
+            foreach ( $images as $image ) {
+                $image['display_src'] = preg_replace( "/^http:/i", "", $image['node']['display_url'] );
+                if ( $image['is_video']  == true ) {
```

Comment:
1324 | ERROR   | [x] Tabs must be used to indent lines; spaces are not allowed
 1324 | WARNING | [ ] Found: ==. Use strict comparisons (=== or !==).
 1324 | ERROR   | [ ] Use Yoda Condition checks, you must.
 1324 | ERROR   | [x] Expected 1 space before "=="; 2 found